scripts/config/icq_config.yaml

```diff
@@ -28,3 +28,31 @@ chains:
     block-timeout: ""
     output-format: json
     sign-mode: direct
+  juno:
+    key: icq3
+    chain-id: JUNO
+    rpc-addr: http://juno1:26657
+    grpc-addr: http://juno1:9090
+    account-prefix: juno
+    keyring-backend: test
+    gas-adjustment: 1.2
+    gas-prices: 1ujuno
```

Comment:
do you know what this param does? default gas-prices argument for relayer-submitted txs, perhaps? This is the first time I really squinted at it

Reply:
I'm not certain either, but I imagine it's along those lines